keyboards/1upkeyboards/sweet16/keymaps/via/keymap.c

```diff
@@ -0,0 +1,60 @@
+#include QMK_KEYBOARD_H
+
+enum custom_keycodes {
+  UP_URL = SAFE_RANGE
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = LAYOUT_ortho_4x4(
+        KC_7, KC_8,   KC_9,   KC_ASTR,
+        KC_4, KC_5,   KC_6,   KC_SLSH,
+        KC_1, KC_2,   KC_3,   KC_MINS,
+        KC_0, KC_ENT, KC_DOT, KC_EQL
+    ),
+
+  [1] = LAYOUT_ortho_4x4(
+        KC_NO,    KC_NO,    KC_NO,    KC_NO,
+        KC_NO,    KC_NO,    KC_NO,    KC_NO,
+        KC_NO,    KC_NO,    KC_NO,    KC_NO,
+        KC_NO,    KC_NO,    KC_NO,    KC_NO
+    ),
+
+  [2] = LAYOUT_ortho_4x4(
+        KC_NO,    KC_NO,    KC_NO,    KC_NO,
+        KC_NO,    KC_NO,    KC_NO,    KC_NO,
+        KC_NO,    KC_NO,    KC_NO,    KC_NO,
+        KC_NO,    KC_NO,    KC_NO,    KC_NO
+    ),
+
+  [3] = LAYOUT_ortho_4x4(
+        KC_NO,    KC_NO,    KC_NO,    KC_NO,
+        KC_NO,    KC_NO,    KC_NO,    KC_NO,
+        KC_NO,    KC_NO,    KC_NO,    KC_NO,
+        KC_NO,    KC_NO,    KC_NO,    KC_NO
+    ),
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+        case UP_URL:
```

Comment:
Does this work in VIA?

Reply:
User defined keycodes aren't in VIA yet. VIA would just show the keycode in hex.

Reply:
So, should it be removed, or?

Reply:
I don't think it matters, it's not referenced in the keymap.